config/inheritOptionFlags.test.coffee

```diff
@@ -0,0 +1,68 @@
+traverse = require 'traverse'
+inherit = require './inheritOptionFlags'
+
+{ expect }  = require '../workers/social/testhelper'
+
+test_options =
+  level1a       :
+    level2a     : 'test-level2a-value'
+    level2b     :
+      level3a   : 'test-level3a-value'
```

Comment:
Could you make this `level3b` with that way we can ensure that we preserve the existing ones while adding a new property in "level 3" items.